app/views/idv/sessions/step1.html.slim

```diff
@@ -0,0 +1,48 @@
+- title t('idv.titles.welcome')
+
+
+.mb3.h6.caps.red = t('idv.titles.welcome')
+= form_tag('/idv/sessions/1', method: 'put')
```

Comment:
since the `1` in `/idv/sessions/1` is a meaningless number just there to satisfy the default Rails routing pattern, maybe we should use a more explicitly meaningless value? E.g. `step2` or `financial-step` or something?